.github/workflows/Connector Master SDK Workflow.yml

```diff
@@ -124,11 +131,13 @@ jobs:
           }      
 
       - name: Building
+        if: steps.detectCsprojFiles.outputs.csproj-file-present == 'true'
```

Comment:
As on Jenkins we will still build the solution due to QAction_Helper, then this condition can be removed.

Reply:
removed